### PR TITLE
Add endpoint to fetch individual and multiple dashboard summaries

### PR DESCRIPTION
**Pull request by sultan-duisenbay**
Add the functionality to fetch individual and multiple dashboard summaries in aggregated format.
This corresponds to data rendered in the dashboard group view - 
image

### Major changes
- Added rpcs to api proto
- Implemented those rpcs
- Added tests

### CODE REVIEW

File: pkg/api/v1/summary_test.go [hunk 1]
```diff
@@ -928,3 +928,324 @@ func TestListTabSummariesHTTP(t *testing.T) {
 		})
 	}
 }
+
+func TestListDashboardSummaries(t *testing.T) {
```
Comment (michelle192837) [inline pkg/api/v1/summary_test.go:932]: nit: Looks good! Maybe a couple more test cases here and below:
- Dashboard group exists but has no dashboards
- Fetching correctly by normalized name
- Fetching correctly by normalized-equivalent name

**Review by michelle192837 — APPROVED** [review]
/hold

For addressing comment

File: pkg/api/v1/summary_test.go (hunk 1 above)
Comment (sultan-duisenbay) [reply pkg/api/v1/summary_test.go:932]: Good feedback! Will add those

Comment (sultan-duisenbay) [reply pkg/api/v1/summary_test.go:932]: Done! PTAL - thanks :)

**Comment by michelle192837** [comment]
LGTM, thanks!

One last thing (can be done as a followup PR), I think the new endpoints can be added to  https://github.com/GoogleCloudPlatform/testgrid/tree/master/pkg/api.

**Pull request merged**